File: src/CSPAF/CSPAgents/CSPUtilityAgents/CSPClickUtilityAgent.cs
using System;
using System.Collections.Generic;

using CloudSuitePortal;
using CloudSuitePortal.Models;
using CloudSuitePortal.Services;
using CloudSuitePortal.Controllers;

namespace CSPAF.CSPAgents.CSPUtilityAgents
{
    class CSPClickUtilityAgent
    {
        public void run(Dictionary<string, object> Params)
        {
            var guid = (string)Params["GUID"];
            var navigationController = CSPBrowser.Application.WindowController.NavigationController;
            var controller = (CSPUtilityAgentsController)navigationController.VisibleViewController;
            controller.ClickAgent(guid);
        }
    }
}

File: src/CSPAF/CSPAgents/CSPUtilityAgents/CSPClickUtilityAgent.java
package CSPAF.CSPAgents.CSPUtilityAgents;

import java.util.*;

import com.infor.cloudsuite.portal.core.services.*;
import com.infor.cloudsuite.portal.core.controllers.*;
import com.infor.cloudsuite.portal.singletenant.agents.controllers.*;

import org.openqa.selenium.NotFoundException;

class CSPClickUtilityAgent
{
    public void run(HashMap<String, Object> params) throws NotFoundException
    {
        String guid = params.get("GUID").toString();
        CSPNavigationController navigationController = CSPBrowser.getApplication().getWindowController().getNavigationController();
        CSPUtilityAgentsController controller = (CSPUtilityAgentsController)navigationController.getVisibleViewController();
        controller.clickAgent(guid);
    }
}